ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -72,6 +72,17 @@ int convertFontWeight (FontWeight fontWeight) {
     }
 } 
 
+TextDecoration convertDecoration (TextDecorationLineType textDecoration){
+    switch(textDecoration){
+        case TextDecorationLineType::None : return TextDecoration::kNoDecoration;
```

Comment:
this case can be clubbed along with default ?

Reply:
Updated